### ISSUE TITLE
Transparency and trust

### ISSUE THREAD

**Issue by Remzi1993** [tenacityteam/tenacity-legacy#121]
I'm very sorry in advance that I need to write this, but I want to get this off my chest.

I was following this project for a while, without being too active within it because I don't know the programming languages (yet) and I'm a web developer, but what I seem to see a lot is deletion of issues instead of closing them for archival purposes. Closing issues breaths transparency and let others verify results and whatnot.

The last straw for me personally is the sudden deletion of the issue: Project Re-branding/Re-naming due to Trademark Issues (#5)

I wanted to see which naming "won" because I was counting the vote (I just got up this morning) and now I couldn't verify anything, because to me AudioBakery had the most votes. But now nobody can't verify anything. This is for me another red flag that the team who is managing this repo is either unfamiliar how huge open source projects are run or might be forcing their will upon the project either case I'm done, will unstar this project and rescind my personal support. I can't support a project without trust.

Trust is a two way street, a dialogue not a monologue. Duo to this I will be looking for another fork to follow and maybe in the future support. This other fork seems to be reasonable: https://github.com/SartoxOnlyGNU/audacium

My apologies and I wish you guys the best.

**Comment by n0toose** [comment]
Hi, I can understand your concerns; maybe explaining the background behind the decision.

AudioBakery was already used by another group of people, that being a space comprising of music producers.

In order to choose the following name, we followed the following procedure that we came up with on a whim, in order to finally choose a name.

- Some people affiliated with Audacium and Tenarity came together in one group and started discussing about the forks, exchanging patches and coming up with a name. We started guessing names, but after a couple of hours, we realized that it was not.
- People that showed interest in the project, including participants from the IRC, a group on Discord, people that were interested in packaging the software, some moderators and two prominent developers that also expressed interest in the project. Cookie did **not** participate in this process and did not influence the final decision for the name or do much else apart from renaming the GitHub organization.
- Many of us started coming up with names and made sure that they were valid.

`After a name is finally chosen, hopefully with some additional participation involved, we'll carefully vet it to see if we can use it. Domain names, social media handles, trademarks, all of that.` Another factor was how easy the name would be to pronounce for ESL speakers.

- After we came up with a couple of names, we came up with the following rules:

```
  1. Voters rank the name by preference on their ballots.
  2. If a name wins an outright majority of first-preference votes (i.e., 50 percent plus one), it wins.
  3. If, on the other hand, no names win an outright majority of first-preference votes, the names with the fewest first-preference votes is eliminated.
  4. All first-preference votes for the failed name are eliminated, lifting the second-preference choices indicated on those ballots.
  5. A new tally is conducted to determine whether any name has won an outright majority of the adjusted votes.
  6. The process is repeated until a candidate wins a majority of votes cast.
```

```
Each person pastes up to three numbers in order of preference (which will better in the event of a tie).
After a name is finally chosen, hopefully with some additional participation involved, we'll carefully vet it to see if we can use it. Domain names, social media handles, trademarks, all of that.
If it doesn't work or if it can't work at all, we'll go for the second best option, so on and so forth.
You can cancel out your previous vote by responding with a new list of three numbers.
```

The names we voted on were the following: `1) Weyvs, 2) Erweck, 3) Audiofect or Audiofekt, 4) AudioBake, 5) SoundMonkey, 6) Audacium, 7) Tenacity, 8) Temerity`

After a long vote, Tenacity and Audacium ended up sort of "tying". Tenacity had just one more vote than Audacium, and we agreed that the order of the preference in the votes was not helpful, so we came up with a new vote between Audacium and Tenacity. Tenacity won. The process also included the scenario where we'd find out that a name was invalid in the sense of actually being trademarked, after all, or already claimed by another group of people that was affiliated with the field of sound editing and music production. In case that happened, we'd select the second most voted preference and talk about it instead.

There were 16 people involved, if I remember correctly. We tried to include as many community members as we could, but we did not overdo it because we feared that if the list of the names got leaked, people would try to claim the usernames on social media platforms and the domains as well.

We believed that choosing the name was basically existential: If we did not have a name chosen, we could not proceed with many other things that are fundamental, such as packaging, documentation or creating chat rooms: Choosing a name was an existential problem for the fork that had to be dealt with, and all of this took us over 10 hours until we came up with this procedure.

We temporarily chose to compromise on transparency after the vote got deleted instead of trying again the same way we did either. I think we can all agree that the initial voting process and the way it was handled did not go very well? We're trying to sort out the governance first, because we really don't want to repeat what we see as Audacity's mistakes. We tried our best to come up with a selective sample that comprised of people that were generally interested in the project with all sorts of different backgrounds - the people that were not directly affiliated with either of these projects was higher than the number of the contributors in there. I personally think that Tenacity as a project should try to integrate the community as much as possible, because there's generally a huge problem with free software developers creating software for other developers, rather than users.

Please understand that we are under a lot of pressure and in the middle of a lot of public attention, and we just tried our best to make this work.

**Comment by cshaa** [comment]
See also #94

**Comment by n0toose** [comment]
I'm generally open to listening to feedback as to how we carried this out and I am also open to listening to questions - there's some details that I did leave out because the post would get way too large if I brought them up. I think that getting the governance straight and the way all of this is going to work is much more important than getting straight to writing code and familiarizing ourselves with the codebase and all of those things, so I was wondering if we should implement this procedure further in the future with a few changes. While some posts did come from people that previously spammed the repository or upvoted other people that did this, some users did get blocked from commenting in the organization unfairly. I can attest to this too, I've even had a friend of mine who directly mocked the people trolling get banned. The volume of things the moderators had to deal with was very large, however, so, maybe we could call this a bumpy start?

I'm very worried about keeping things open to the general public. Do you (or anyone else) think we should establish some sort of system in order to establish who's a trusted community member or isn't with very lax requirements, or follow the [Arch Linux model of governance](https://wiki.archlinux.org/title/Trusted_Users), which is relatively more strict?

I am not the person "in charge" or the "benevolent dictator" for the day, nor do I want to be after the whole drama that went down, but I think it's safe to say that every single one of us that are labelled as "Contributors" really just want to split the power as much as possible among as many people as possible to a reasonable degree. Personally, I don't think that my C++ knowledge is good enough to take on with developing an entire Audacity fork. In all honesty, I just wanted to write the code, learn and contribute to something great, not write public service announcements. You can join us on `#tenacity` on Libera too. Keep in mind that the chat is being monitored by the network's staff members for violations of their Terms of Service.

**Comment by n0toose** [comment]
We gave a very prominent member of the free and open-source software community that seemed very invested in the well-being of the project and two relevant GitHub contributors a vote as well, but I'm not naming any names right now out of respect for their (just like everyone else's) privacy. I'll be forwarding this post to them in case they want to vouch for us.

But yeah, most of us just want to write and do something great and meaningful here, not go on a power trip. It's not like every other project tries to start off with a sum total of 2.8k stars on GitHub *or* that there are people that are willing or capable of accepting this level of responsibility.

**Comment by n0toose** [comment]
I should also note that the entire 'Member' and 'Collaborator' thing is sort of arbitrary - people that were here from the start and others that are currently helping organize things, so that there isn't this large number of people just doing stuff by themselves in a completely uncontrollable manner. (But we do need more people, that's for sure.)

People on here, including myself, may only stay here temporarily. The people that are basically on board right now are doing their jobs quite well and I hope that they'll eventually prove themselves to you and others, but, say, as far as I am concerned, I was very afraid (and still am) to even show my presence out here because of everything that's currently happening with and around the person who initiated the project. I hope that all of us will hopefully set the criteria for promoting people and assigning responsibilities soon.

As far as I am concerned, I don't really have any vested interests or reason to contribute here rather than other projects. I'm just a kid that likes programming cool stuff, honestly.

**Comment by n0toose** [comment]
For now, however, we'll need some time off, as the past two days have been very challenging. I'll be here to answer questions if you need me to when I decide to look at GitHub again.

Afterwards, we'll start dealing with [the stuff that should be done after we choose the name.](https://github.com/tenacityteam/tenacity/projects) Anyone can help, again, over at `#tenacity` on [Libera](https://libera.chat).

**Comment by awslabspl** [comment]
> For now, however, we'll need some time off, as the past two days have been very challenging. I'll be here to answer questions if you need me to when I decide to look at GitHub again.
> 
> Afterwards, we'll start dealing with [the stuff that should be done after we choose the name.](https://github.com/tenacityteam/tenacity/projects) Anyone can help, again, over at `#tenacity` on [Libera](https://libera.chat).

Please see below
![Zrzut ekranu 2021-07-7 o 11 07 27](https://user-images.githubusercontent.com/77433061/124732589-9f9a6300-df13-11eb-9240-eb48b1dfe449.png)
![Zrzut ekranu 2021-07-7 o 11 07 54](https://user-images.githubusercontent.com/77433061/124732595-a0cb9000-df13-11eb-8a41-96642d08e691.png)

**Comment by n0toose** [comment]
> Please see below

Hi, I'm not sure why this could be happening. I'll ask around and message you via IRC when I find out. (I suggest you do the same.) I edited your message so that the screenshots redirect to a link instead of a full-on image in order to keep things clean here, I hope that this is okay.

**Comment by Sciss** [comment]
what I did is follow the steps here: https://superuser.com/questions/106965/how-do-i-identify-with-services-on-irc

**Comment by Be-ing** [comment]
I appreciate the detailed response @panos. I also appreciate that there were extraordinary circumstances that warranted the privacy away from the onslaught of trolls.

**Comment by nova-nowiz** [comment]
Thank you very much for this long post explaining the situation @panos!

**Comment by awslabspl** [comment]
As of transparency::

May someone enlight me what was wrong in me asking ( on IRC ) to be maintainer? Especially in the light of #99 ?

**Comment by n0toose** [comment]
> May someone enlight me what was wrong in me asking ( on IRC ) to be maintainer?

For the record, you were asking that we should give you full read and write access on the repository without any prior involvement or justification other than "because I am myself". I can paste the IRC logs if you want to make a big deal out of this as long as you also agree to do so. I'll mark both of these replies as off-topic, because they pretty are, I think. Nothing is wrong with doing so, but flaming us for "doing what other projects do" and for asking for proof of worth when no project really gives full access to a person as soon as they ask is really not cool. :/

If anyone has any further questions, do reach out.

**Comment by awslabspl** [comment]
@panos First: I dont understand why this is offtopic, but Im ok with it; if you ( or someone else ) wants it: let it be
Second: I thought that, after I expressed willingness to take the role of maintainer, you will go to my profile's page here on GH and find out for yourself........ some 80% of my contribs are public. I thought that wouldnt be a problem for you.
Third: Im OK with you posting logs of IRC here, have nothing to hide.

**Comment by Semisol** [comment]
> @panos First: I dont understand why this is offtopic, but Im ok with it; if you ( or someone else ) wants it: let it be
> Second: I thought that, after I expressed willingness to take the role of maintainer, you will go to my profile's page here on GH and find out for yourself........ some 80% of my contribs are public. I thought that wouldnt be a problem for you.
> Third: Im OK with you posting logs of IRC here, have nothing to hide.
> 
> What Im **NOT OK** is the way Ive been denied. Not that I wouldnt accept denial, but I **will not** accept it expressed the way it was expressed. if you dont want me on the team, that perfectly fine, but communicate it to me in the civilized manner.....

@awslabspl 
I think I expressed my denial in a correct way. Can you tell me what offends you?
Your reasonings do not make sense (counterarguments):
1. "because I am me" You are not different than others, until you show what makes your different.
2. "because I have the experience to maintain a big fork" Prove it. Your (no offense) immature and partially informal actions sort of make it apparent. (I'd expect people to at least be formal and not act immature if they really are willing to work for a project)

And denial reasons:
1. You have not provided any valid reasoning.
	- You constantly repeating rejected reasonings makes us not want to do it
2. Your begging-like actions
3. Not participating in the community enough to be considered

So now, please do not extend this discussion if you do not have any more constructive things to say.

**Comment by awslabspl** [comment]
> Prove it.

@Semisol visit my public GH profile............

**Comment by Remzi1993** [comment]
> Hi, I can understand your concerns; maybe explaining the background behind the decision.
> 
> AudioBakery was already used by another group of people, that being a space comprising of music producers.
> 
> In order to choose the following name, we followed the following procedure that we came up with on a whim, in order to finally choose a name.
> 
> * Some people affiliated with Audacium and Tenarity came together in one group and started discussing about the forks, exchanging patches and coming up with a name. We started guessing names, but after a couple of hours, we realized that it was not.
> * People that showed interest in the project, including participants from the IRC, a group on Discord, people that were interested in packaging the software, some moderators and two prominent developers that also expressed interest in the project. Cookie did **not** participate in this process and did not influence the final decision for the name or do much else apart from renaming the GitHub organization.
> * Many of us started coming up with names and made sure that they were valid.
> 
> `After a name is finally chosen, hopefully with some additional participation involved, we'll carefully vet it to see if we can use it. Domain names, social media handles, trademarks, all of that.` Another factor was how easy the name would be to pronounce for ESL speakers.
> 
> * After we came up with a couple of names, we came up with the following rules:
> 
> ```
>   1. Voters rank the name by preference on their ballots.
>   2. If a name wins an outright majority of first-preference votes (i.e., 50 percent plus one), it wins.
>   3. If, on the other hand, no names win an outright majority of first-preference votes, the names with the fewest first-preference votes is eliminated.
>   4. All first-preference votes for the failed name are eliminated, lifting the second-preference choices indicated on those ballots.
>   5. A new tally is conducted to determine whether any name has won an outright majority of the adjusted votes.
>   6. The process is repeated until a candidate wins a majority of votes cast.
> ```
> 
> ```
> Each person pastes up to three numbers in order of preference (which will better in the event of a tie).
> After a name is finally chosen, hopefully with some additional participation involved, we'll carefully vet it to see if we can use it. Domain names, social media handles, trademarks, all of that.
> If it doesn't work or if it can't work at all, we'll go for the second best option, so on and so forth.
> You can cancel out your previous vote by responding with a new list of three numbers.
> ```
> 
> The names we voted on were the following: `1) Weyvs, 2) Erweck, 3) Audiofect or Audiofekt, 4) AudioBake, 5) SoundMonkey, 6) Audacium, 7) Tenacity, 8) Temerity`
> 
> After a long vote, Tenacity and Audacium ended up sort of "tying". Tenacity had just one more vote than Audacium, and we agreed that the order of the preference in the votes was not helpful, so we came up with a new vote between Audacium and Tenacity. Tenacity won. The process also included the scenario where we'd find out that a name was invalid in the sense of actually being trademarked, after all, or already claimed by another group of people that was affiliated with the field of sound editing and music production. In case that happened, we'd select the second most voted preference and talk about it instead.
> 
> There were 16 people involved, if I remember correctly. We tried to include as many community members as we could, but we did not overdo it because we feared that if the list of the names got leaked, people would try to claim the usernames on social media platforms and the domains as well.
> 
> We believed that choosing the name was basically existential: If we did not have a name chosen, we could not proceed with many other things that are fundamental, such as packaging, documentation or creating chat rooms: Choosing a name was an existential problem for the fork that had to be dealt with, and all of this took us over 10 hours until we came up with this procedure.
> 
> We temporarily chose to compromise on transparency after the vote got deleted instead of trying again the same way we did either. I think we can all agree that the initial voting process and the way it was handled did not go very well? We're trying to sort out the governance first, because we really don't want to repeat what we see as Audacity's mistakes. We tried our best to come up with a selective sample that comprised of people that were generally interested in the project with all sorts of different backgrounds - the people that were not directly affiliated with either of these projects was higher than the number of the contributors in there. I personally think that Tenacity as a project should try to integrate the community as much as possible, because there's generally a huge problem with free software developers creating software for other developers, rather than users.
> 
> Please understand that we are under a lot of pressure and in the middle of a lot of public attention, and we just tried our best to make this work.

Thank you very much for your detailed answer, however there are things I struggle with because I also would have liked to be involved (like many others). We Europeans and people from other parts of the world don't share the same time zone. I assumed this all happened very quickly when I was sleeping - which is strange before giving a heads up or maybe planning a date and time to vote for a name.

If I'm completely honest - (and I hope I don't offended someone) my apologies if you get offended in advance - I really don't like the new name/branding. I'm very sorry, but I would like to point out that there were also other secondary names out there in the discussed issue I was referring to. Now that the issue is closed I can't point to same examples. If I think hard, SoundOven also had a lot of votes and some other names as well.

I don't know were the new name is referring too or even means? Is it a (word) joke to Audacity? Tenacity? I don't find it original. No offense and hard feelings. People have different time zones and such and it would have been nice to include everyone who is either participating/contributing or wants to support this project in the (near) future. I'm sorry to tell this, but I won't be supporting this project. I can't work like this, suddenly out of the blue something is decided elsewhere on Discord, IRC, whatever and something happens here. I don't know the channels and quite frankly keep things separated especially work, development and fun (I use Discord for gaming with friends and whatnot).

My apologies if I offended someone, I just wanted to get this off my chest before quitting.

**Comment by n0toose** [comment]
> I assumed this all happened very quickly when I was sleeping - which is strange before giving a heads up or maybe planning a date and time to vote for a name.

No, you're right, we did not announce this because we were worried about infiltrators and reached out to people talking about all of this. The decision about which people are going to be chosen were random, e.g. I included one person who asked me about a GitHub issue.

> If I'm completely honest - (and I hope I don't offended someone) my apologies if you get offended in advance - I really don't like the new name/branding.

Yeah, I guess nobody could possibly be happy with any of the decisions for the name and the branding, even if the name would've been chosen with the help of the public. I don't think that many other projects leave decisions like this (especially without a solid framework of rules) up to the public, because of, well, reasons that should be obvious by now.

> I don't know were the new name is referring too or even means?

When I found out about Audacity, I personally didn't know what the word meant in English either. I guess this is a similar situation? I guess that it's meaning is meant to represent the project's goals while paying respect to its roots.

> People have different time zones and such and it would have been nice to include everyone who is either participating/contributing or wants to support this project in the (near) future.

> include everyone who is either participating/contributing or wants to support this project in the (near) future.

> I can't work like this, suddenly out of the blue something is decided elsewhere on Discord, IRC, whatever and something happens here.

It was a one-time thing for a very specific thing, really. I don't think we could've done this the other way around, unless if we disqualified the one entry, but the thread had already been deleted. We brought up some of the names on there, some of which were rejected based on the criteria I brought up earlier, but we could not remember all of them, I guess.

In retrospect, the rest of us could've looked for re-uploaded versions of the names in other places. I can definitely agree with you, but I think you're a bit tougher on us considering that we couldn't have carried this out in public after what happened. Cookie started the vote and I don't think any of the few other people affiliated at the time got the time to think stuff like this through.

> (I use Discord for gaming with friends and whatnot).

The Discord server belonged to Audacium. The project does not officially use Discord on our channels, but we found interested people that way. I also use Discord the same way, although I generally avoid from using the platform, and I personally don't think that it's a good platform for developing such software.

> I'm sorry to tell this, but I won't be supporting this project.

I also distanced myself from the project indefinitely. but I'm just saying this as a person who was a part of that group and knows what happened. I can't just say anything more than that we tried our best to find the fine line between including people and a low amount of risk, so that a single person and a few other people wouldn't decide that sort of future. I guess I'm sorry that what we did did not satisfy you, I didn't like how it went down either. It's just that all of the people that played a role in this were under immensely pressuring, very stressful and unforeseeable circumstances that greatly delayed our project and affected our ability to coordinate, that is all.

Now that everything has sort of calmed down, I do hope that things will be considerably more transparent from this point on, and the groundwork required to achieve this has been laid out already. Stay safe.
